beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/model/EntityHousekeepingPath.java

```diff
@@ -52,7 +52,7 @@
 
   @Column(name = "path_status", nullable = false)
   @Enumerated(EnumType.STRING)
-  private PathStatus pathStatus;
+  private HousekeepingStatus pathStatus;
```

Comment:
Was the field name changed to match the interface? The column annotation will map the field to the correct column (`path_status`) in the table, but it does look a bit weird. Perhaps we could run a table migration to alter the name of this column from `path_status` to `housekeeping_status`?

Reply:
I'm not too bothered if it stays pathStatus in the table what do others think?

Reply:
I'd be up for renaming the column if it's not too much effort to roll that out.